types/kagekiri.d.ts

```diff
@@ -1,3 +1,4 @@
 // Type definitions for kagekiri
 export function querySelector(selector: string, context?: Node): HTMLElement | null;
-export function querySelectorAll(selector: string, context?: Node): NodeList;
\ No newline at end of file
+export function querySelectorAll(selector: string, context?: Node): NodeList;
+export function getElementsByClassName(names: string, context?: Node): NodeList;
```

Comment:
re `NodeList` I don't remember what we decided on but it felt weird to type it as an `HTMLCollection` when it specifically isn't.

Reply:
It's also technically not a `NodeList` – it's just an Array of `Node`s. I mean, for most intents and purposes, it's the same, but there are subtle differences apparently: https://developer.mozilla.org/en-US/docs/Web/API/NodeList

Reply:
Hmmm on the other hand, if we identify it as an HTMLCollection, at least implementors can observe and handle that accordingly.

Reply:
Actually yeah then it should just be a `Node[]`